tests/auth_tests/test_management.py

```diff
@@ -1571,16 +1572,12 @@ def assertOperationsInjected(self, plan, **kwargs):
                     self.assertEqual(next_operation.new_model, operation.new_name)
 
     def test_permission_rename(self):
-        ct = ContentType.objects.create(app_label="auth_tests", model="oldmodel")
-        actions = ["add", "change", "delete", "view"]
-        for action in actions:
-            Permission.objects.create(
-                codename=f"{action}_oldmodel",
-                name=f"Can {action} old model",
-                content_type=ct,
-            )
+        # Create initial content type and permissions for OldModel.
+        call_command("migrate", "auth_tests", "0001", verbosity=0)
```

Comment:
Note for next week: need to add a test that migrates from zero to 0002 in one shot to ensure the result does not differ, given there are no ContentType records when the pre_migrate signal fires.